stuff/cron/testing/test_plagiarism_detector.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+from typing import List
+import pytest
+import argparse
+import calendar
+import collections
+import datetime
+import json
+import logging
+import string
+import time
+import operator
+import sys
+import unittest
+import os
+import random
+
+sys.path.insert(
+    0,
+    os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
+                "../"))
+
+
+import lib.db
+import omegaup.api
+import test_constants
+from plagiarism_detector import get_contests
+
+UPDATE_CHECK_PLAGIARISM = '''
+                            UPDATE Contests
+                            SET check_plagiarism = 1
+                            WHERE alias = %s;
+
+'''
+
+@pytest.fixture(scope='session')
+def dbconn() -> lib.db.Connection:
+    dbconn = lib.db.connect(
+        lib.db.DatabaseConnectionArguments(
+            user='root',
+            password='omegaup',
+            host='mysql',
+            database='omegaup',
+            port= 13306,
+            mysql_config_file=lib.db.default_config_file_path() or ''
+        )
+    )
+    return dbconn
+
+def test_get_contests(dbconn: lib.db.Connection) -> None:
+
+    
+    client = omegaup.api.Client(api_token=test_constants.API_TOKEN,
+                                url=test_constants.OMEGAUP_API_ENDPOINT)
+    current_time = datetime.datetime.now() - datetime.timedelta(minutes=30)
+    future_time = datetime.datetime.now()
+    alias = ''.join(random.choices(string.digits, k=8))
+
+    # Creating a contest and then adding some users
+    client.contest.create(
+        title=alias,
+        alias=alias,
+        description='Test contest',
+        start_time=time.mktime(current_time.timetuple()),
+        finish_time=time.mktime(future_time.timetuple()),
+        window_length=0,
+        scoreboard=100,
+        points_decay_factor=0,
+        partial_score=True,
+        submissions_gap=1200,
+        penalty=0,
+        feedback='detailed',
+        penalty_type='contest_start',
+        languages='py2,py3',
+        penalty_calc_policy='sum',
+        admission_mode='private',
+        show_scoreboard_after=True,
+    )
+    with dbconn.cursor(dictionary=True) as cur:
+        cur.execute(UPDATE_CHECK_PLAGIARISM, (alias,))
```

Comment:
```suggestion
        cur.execute(UPDATE_CHECK_PLAGIARISM, (alias,))
        dbconn.commit()
```

note that python's sql implementation does not autocommit, so we need to do that manually every time!

Reply:
Ohhh yess!!! Thanks for making me remember.